pkg/export/template.go

```diff
@@ -76,9 +76,7 @@ func (t *Template) parseTemplate(tpl string) (*template.Template, error) {
 	return template.New("").Funcs(templateFuncs).Parse(tpl)
 }
 
-func (t *Template) Execute(input io.Reader) error {
-	w := t.io.Out
-
+func (t *Template) Execute(w io.Writer, input io.Reader) error {
 	if t.template == nil {
 		template, err := t.parseTemplate(t.templateStr)
```

Comment:
Since `parseTemplate` uses `t.io` to determine if the output should be colored, it seems like there could be a case there the writer passed to `Execute` does not support colors where the one used when initializing the template does leading to weird output. This seems like a rare edge case though.

Reply:
Good point. This could definitely be improved. Right now, I just wanted to be able to temporarily override the output writer without having to mutate the IOStreams object for this.